scripts/csv.addColumn.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
```

Comment:
Accidental commit?

Reply:
yes, i forgot to delete it

Reply:
after talking in discord with you and everyone, i left the scripts in. It might make sense to add scripts like "deleteColumn" etc. for general usage